ui/src/routes/Units/Units.js

```diff
@@ -10,6 +12,23 @@ import '../../styles/buttons.css'
 let backgroundImage = require('../../assets/clix-i2c-flowers.svg')
 
 class Units extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      skiplinks: [{
```

Comment:
Do you need to define `skipLinks` here? I think it's already defined in `PageFocusSection`...

Reply:
Ah, yes. I'll pull these extraneous skiplink state definitions out. Also found stray prop validation warning. Thanks for the catch!